intercom_flutter/example/lib/main.dart

```diff
@@ -9,6 +9,7 @@ void main() async {
     androidApiKey: 'androidApiKey',
     iosApiKey: 'iosApiKey',
   );
+  // TODO: don't forget to set up the custom application class on Android side.
```

Comment:
Not really a TODO
```suggestion
  Don't forget to set up the custom application class on Android side.
```

Reply:
TODO is added because then the comment will highlight